minigeth/core/types/batch_tx.go

```diff
@@ -0,0 +1,74 @@
+package types
+
+import (
+	"math/big"
+
+	"github.com/ethereum/go-ethereum/common"
+)
+
+type BatchTx struct {
+	ChainID       *big.Int
+	DecryptionKey []byte
+	BatchIndex    uint64
+	L1BlockNumber *big.Int
+	Timestamp     *big.Int
+	Transactions  [][]byte
+
+	// Signature values
+	V *big.Int `json:"v" gencodec:"required"`
+	R *big.Int `json:"r" gencodec:"required"`
+	S *big.Int `json:"s" gencodec:"required"`
+}
+
+// copy creates a deep copy of the transaction data and initializes all fields.
+func (tx *BatchTx) copy() TxData {
```

Comment:
looks like the signature fields are missing here

Reply:
good catch

Reply:
Fixed